include/tcframe/io_manipulator/GridIOSegmentManipulator.hpp

```diff
@@ -17,10 +17,14 @@ namespace tcframe {
 
 class GridIOSegmentManipulator {
 public:
-    static void parse(GridIOSegment* segment, istream* in) {
+    static string parse(GridIOSegment* segment, istream* in) {
```

Comment:
For each segment manipulator, you need to add test(s) to verify that the return value (last variable name) is correct. Don't modify existing tests -- add new tests instead that just do that.